src/stabby/fatptr_impl.rs

```diff
@@ -0,0 +1,91 @@
+use crate::{
+    derive_ReprC,
+    ඞ::{CLayoutOf, ConcreteReprC, ReprC},
+};
+use stabby::abi::{vtable::HasDropVt, IPtrOwned};
+
+unsafe impl<Ptr: ConcreteReprC + IPtrOwned + ReprC, VTable: HasDropVt + ReprC> ReprC
+    for stabby::abi::Dyn<'_, Ptr, VTable>
+{
+    type CLayout = CLayoutOf<StabbyDyn<Ptr, VTable>>;
+    fn is_valid(it: &'_ Self::CLayout) -> bool {
+        StabbyDyn::<Ptr, VTable>::is_valid(it)
+    }
+}
+
+unsafe impl<'a, VTable: HasDropVt + ReprC> ReprC for stabby::abi::DynRef<'a, VTable> {
+    type CLayout = CLayoutOf<StabbyDyn<&'a (), VTable>>;
+    fn is_valid(it: &'_ Self::CLayout) -> bool {
+        StabbyDyn::<&'a (), VTable>::is_valid(it)
+    }
+}
+
+#[derive_ReprC]
+#[repr(C)]
+#[derive(Debug)]
+pub struct StabbyDyn<Ptr, VTable: 'static> {
+    pub data: Ptr,
+    pub vtable: &'static VTable,
+}
+
+#[derive_ReprC]
+#[repr(C)]
+#[derive(Debug, Clone, Copy)]
+pub struct VtDrop {
+    pub v_drop: extern "C" fn(*const ()),
```

Comment:
Very tiny nit, even if it doesn't change anything in practice 😄:

```suggestion
    pub v_drop: unsafe extern "C" fn(*const ()),
```